src/core/api.proto

```diff
@@ -223,6 +223,25 @@ message InferRequestHeader
   //@@     request.
   //@@
   repeated Output output = 3;
+
+  //@@  .. cpp:var:: uint32 priority [DLIS-1083]
+  //@@
+  //@@     The priority value of this request. If the priority value is not
+  //@@     enable by the model, then the request will not be prioritized.
+  //@@     The default value is 0 which indicates that the request will not be
+  //@@     prioritized.
+  //@@
+  uint32 priority = 6;
+
+  //@@  .. cpp:var:: uint64 timeout_limit_microseconds [DLIS-997]
+  //@@
+  //@@     The timeout limit specified for this request. This value will
```

Comment:
"The timeout for this request. This value overrides the timeout specified by the model, if the model allows timeout override. If the request cannot be processed within this timeout, the request will be handled based on the model's timeout policy."